docs/_docs/animations.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Animations
+layout: page
+nav_order: 13
+---
+
+# Animations
+
+Animations are supported in two different ways: animations from Figma interactions, and animations from programmatically changing variants. To enable animations, the squoosh rendering path must be enabled by turning it on and wrapping your `@Composable` content with it:
+```kotlin
+    CompositionLocalProvider(LocalDesignDocSettings provides DesignDocSettings(useSquoosh = true)) {
+        content()
+    }
+```
+
+## Figma interaction animations
+Animations set in Figma as a prototype interaction are supported. These animations run automatically whenever an interaction triggers the animation to begin. After making any changes to an interaction, make sure to run the `Sync Prototype changes` plugin so that the interaction is saved to the plugin data.
+
+<img src="./animations/SyncPrototype.png">
+
+## Variant change animations
+When variants change programmatically using the `@DesignVariant` customization, they will use a default spring animation for the transition. This can be customized using the custom animation API by specifying an easing type and duration in the `DesignDocSetting()` function that enables the squoosh animation code path. Populate the new `customVariantTransition` parameter with a function that takes a `VariantTransitionContext` and returns an `AnimationTransition` interface. The context provides some data on variant that is changing, and the `AnimationTransition` needs a function `animationSpec()`` that returns `AnimationSpec<Float>``. The `SmartAnimationTransition` is the only supported transition at this time. The easiest way to use this is to use a compose function such as `androidx.compose.animation.core.tween`.
```

Comment:
```suggestion
When variants change programmatically using the `@DesignVariant` customization, they will use a default spring animation for the transition. This can be customized using the custom animation API by specifying an easing type and duration in the `DesignDocSetting()` function that enables the squoosh animation code path. Populate the new `customVariantTransition` parameter with a function that takes a `VariantTransitionContext` and returns an `AnimationTransition` interface. The context provides some data on variant that is changing, and the `AnimationTransition` needs a function `animationSpec()` that returns `AnimationSpec<Float>`. The `SmartAnimationTransition` is the only supported transition at this time. The easiest way to use this is to use a compose function such as `androidx.compose.animation.core.tween`.
```

Reply:
I am still seeing the double `` after animationSpec() and AnimationSpec<Float>

Reply:
Oops I force pushed without getting the changes. Fixed now.